src/server.js

```diff
@@ -68,7 +68,7 @@ async function createServer() {
 
   configCleanup(db, logger)
 
-  server.listen(serverConfig.port, () => {
+  server.listen(process.env.PORT || serverConfig.port, () => {
```

Comment:
I would expect the right port to be loaded in serverConfig.port already. Why is this needed?

Reply:
An error occurred when trying to reassign `serverConfig.port` since the`serverConfig` object was not writable. If we don't use a separate config file for the backup instance, the port number will be the same for both instances and we need to somehow make sure they will listen on different ports.